rsocket-core/src/main/java/io/rsocket/plugins/SafeCompositeRequestInterceptor.java

```diff
@@ -0,0 +1,64 @@
+package io.rsocket.plugins;
+
+import io.netty.buffer.ByteBuf;
+import io.rsocket.frame.FrameType;
+import reactor.core.publisher.Operators;
+import reactor.core.publisher.SignalType;
+import reactor.util.context.Context;
+
+public class SafeCompositeRequestInterceptor implements RequestInterceptor {
```

Comment:
Perhaps just `CompositeRequestInterceptor` as the name? I'm not sure it needs to have `Safe` in the name and there are no other (unsafe) implementation.

Reply:
Fixed